tests/tests_pytorch/lite/test_lite.py

```diff
@@ -400,6 +401,8 @@ def test_autocast():
     lite._precision_plugin.forward_context().__exit__.assert_called()
 
 
+# https://github.com/microsoft/DeepSpeed/issues/2139
+@pytest.mark.skipif(_RequirementAvailable("deepspeed>=0.6.5"), reason="Lite does not support 0.6.5")
```

Comment:
@awaelchli We should also have a version check in the Lite code so that users don't think we have bugs.

Reply:
Added that. Note that the test changes I made require your other PR that fixes the Lite test for <0.6.5